src/notation/view/notationnavigator.h

```diff
@@ -36,6 +41,21 @@
 #include "abstractnotationpaintview.h"
 
 namespace mu::notation {
+class NotationNavigatorViewRect : public QQuickPaintedItem
+{
+    Q_OBJECT
+
+    INJECT(INotationConfiguration, configuration)
+
+    virtual void paint(QPainter* painter) override;
+    RectF m_cursorRect;
+
+public:
+    NotationNavigatorViewRect(QQuickItem* parent = nullptr);
+
+    void setRect(RectF cursorRect);
```

Comment:
Code style: we prefer to have public declarations above private ones
```suggestion
public:
    NotationNavigatorViewRect(QQuickItem* parent = nullptr);

    void setRect(RectF cursorRect);

private:
    void paint(QPainter* painter) override;
    
    RectF m_cursorRect;
```